### ISSUE TITLE
Does not work with latest yang-js

### ISSUE THREAD

**Issue by anisse** [corenova/yang-swagger#5]
Hi,

It seems that yang-swagger does not work anymore with the latest yang-js

```
$ ./bin/yang-swagger -f yaml -o swagger.yaml yang-openapi
TypeError: res.spec.serialize is not a function
    at /home/anisse/yangswagger/yang-swagger/bin/yang-swagger:34:25
```
I was looking forward to test the fixed choice handling of yang-js 0.24.45; any idea what changed ?

**Comment by sekur** [comment]
Hmm, there's been some significant refactoring done on yang-js recently. I'll take a look and update `yang-swagger` to restore compatibility. Thanks for letting me know!

**Comment by sekur** [comment]
@anisse should be fixed now (published 2.3.4), please reopen if issue persists.

**Comment by anisse** [comment]
Thanks, it works well.
